src/audio/mux/mux.c

```diff
@@ -530,15 +530,18 @@ static int mux_process(struct processing_module *mod,
 	int frames;
 	int sink_bytes;
 	int source_bytes;
-	int i;
+	int i, j;
 
 	comp_dbg(dev, "mux_process()");
 
 	/* align source streams with their respective configurations */
+	j = 0;
+	frames = INT32_MAX;
```

Comment:
if we do this, then we'd some sanity check for frames before processing

Reply:
in case dev->bsource_list is empty

Reply:
Yep, need to handle that.